src/status_im/translations/es.cljs

```diff
@@ -54,254 +71,295 @@
    :message                               "Mensaje"
-   :username                              "Nombre de usuario"
    :not-specified                         "No especificado"
    :public-key                            "Clave pública"
    :phone-number                          "Número de teléfono"
-   :email                                 "Correo electrónico"
-   :profile-no-status                     "Sin estado"
+   :update-status                         "Actualiza tu estado..."
+   :add-a-status                          "Añade un estado..."
+   :status-prompt                         "Establece tu estado. Usando #hastags permitirás que los demás te encuentren y podrás hablar sobre lo que tengas en mente"
    :add-to-contacts                       "Añadir a contactos"
-   :error-incorrect-name                  "Por favor, selecciona otro nombre"
-   :error-incorrect-email                 "Correo electrónico incorrecto"
+   :in-contacts                           "En contactos"
+   :remove-from-contacts                  "Eliminar de contactos"
+   :start-conversation                    "Empezar conversación"
+   :send-transaction                      "Enviar transacción"
+   :testnet-text                          "Estás en la red de prueba {{testnet}}. No envíes ETH o SNT real a tu dirección"
+   :mainnet-text                          "Estás en la red principal. Se enviará ETH real"
 
    ;;make_photo
-   :image-source-title                    "Imagen de perfil"
-   :image-source-make-photo               "Capturar"
+   :image-source-title                    "Imágen de perfil"
+   :image-source-make-photo               "Captura"
    :image-source-gallery                  "Seleccionar de la galería"
-   :image-source-cancel                   "Cancelar"
 
    ;;sharing
-   :sharing-copy-to-clipboard             "Copiar"
+   :sharing-copy-to-clipboard             "Copiar al portapapeles"
    :sharing-share                         "Compartir..."
    :sharing-cancel                        "Cancelar"
 
-   ;sign-up
-   :contacts-syncronized                  "Se han sincronizado tus contactos"
-   :confirmation-code                     (str "¡Gracias! Te hemos enviado un mensaje de texto con un código"
-                                               "de confirmación. Por favor, introducelo para confirmar tu número de teléfono")
-   :incorrect-code                        (str "Lo sentimos, el código era incorrecto. Por favor escríbelo de nuevo.")
-   :generate-passphrase                   (str "Generaré una frase de respaldo para que puedas restaurar tu contraseña"
-                                               "o iniciar sesión desde otro dispositivo")
-   :phew-here-is-your-passphrase          "*Uf* ha sido difícil, aquí está tu contraseña, * escríbela y ¡guárdala en un sitio seguro! * La necesitarás para recuperar tu cuenta."
+   :browsing-title                        "Navegar"
+   :browsing-browse                       "@navegar"
```

Comment:
please change this back to @browse as it is just a formalism our team have decided on.